task1/lib/main.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Task 1',
+      home: MainPage(),
+    );
+  }
+}
+
+class MainPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Center(
+          child: Text("Flutter Vadodara"),
+        ),
+      ),
+      body: Column(
+        children: [
+          Container(
+            color: Colors.blue,
+            padding: EdgeInsets.fromLTRB(80, 10, 10, 10),
```

Comment:
I don't want so much padding. Read the docs. I want 16 paddings for only top and bottom and 32 margins from left and right. 

Instead of giving padding and margin in every container, you can wrap you `Column` in a `Container` to give padding and margin.